cmd/kubelet/app/server.go

```diff
@@ -211,6 +212,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ResourceContainer, "resource-container", s.ResourceContainer, "Absolute name of the resource-only container to create and run the Kubelet in (Default: /kubelet).")
 	fs.StringVar(&s.CgroupRoot, "cgroup_root", s.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
 	fs.StringVar(&s.ContainerRuntime, "container_runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.")
+	fs.IntVar(&s.MaxPods, "max_pods", 32, "Number of Pods that can run on this Kubelet.")
```

Comment:
I wonder if we should set this higher for now (like 200 or something) since we're doing experiments where we run lots of pods per node, and it would be annoying to have to change the flag default.